app/services/slack_service.rb

```diff
@@ -2,7 +2,17 @@ class SlackService
   def initialize(client = Slack::Client.new)
     @client = client
   end
+
   def post_message(channel, message)
     @client.chat_postMessage(channel: channel, text: message, as_user: true)
   end
+
+  def post_direct_message(token, users, message)
```

Comment:
Like the name here.

Reckon we can get the token from the settings directly from inside this method rather than passing it in? I'm not sure if I'm seeing an advantage to passing it in from the controller. Then we can get rid of the `token` argument, too. Perhaps having that flexibility would be good if we were working with more than one Slack workspace (if they each have a different token). I don't think we're near that stage yet, though.